src/main/resources/templates/email/feedbackToInterviewee.txt

```diff
@@ -0,0 +1,26 @@
+Thank you for practicing your interviewing skills using InterviewMe on {{formatted_date}}. Your interviewer has submitted feedback for your interview which you can view below. 
+
+The values 1-10 represent whether your interviewer strongly disagreed(1) or agreed(10) with the prompt. 
+
+The interviewee communicated their thought process as they went along: {{question_1}}
+The interviewee understood the time complexity of their solution: {{question_2}}
+The interviewee took the time to consider better solutions: {{question_3}}
+The interviewee came up with an example that they used to test their solution: {{question_4}}
+The interviewee listed a handful of edge cases and accounted for their behaviour: {{question_5}}
+The interviewee asked clarifying questions: {{question_6}}
+This InterviewMe experience was helpful: {{question_7}}
```

Comment:
lf we're going with inputs on the next line on the form we should do that here too. So:
[statement]
[answer]
[new line]

Same for other file.

Reply:
This format reflects the current format of the forms.